src/components/organisms/AssetContent/index.tsx

```diff
@@ -46,8 +47,9 @@ export default function AssetContent(props: AssetContentProps): ReactElement {
   const { owner, isInPurgatory, purgatoryData } = useAsset()
   const [showPricing, setShowPricing] = useState(false)
   const [showEdit, setShowEdit] = useState<boolean>()
-  const { ddo, price, metadata } = useAsset()
+  const { ddo, price, metadata, type } = useAsset()
   const isOwner = accountId === owner
+  const accessType = ddo.service[1].type
```

Comment:
right, was seeing that we already have this in `feature/compute` but worth tweaking that in here. While this works right now with assets published through our UI, we should not rely on the `service` array order. And to make it more fun, in there, there could be both, `access` & `compute` service but we can almost always assume that if there's a compute service there should be no download possible.

Not sure about best logic here:
- get the value of the `type` key in one of the `service` objects, where the value is either `access` or `compute`, that seems complicated
- just rely on checking for compute service as a boolean, [as we do in `main`](https://github.com/oceanprotocol/market/blob/main/src/components/organisms/AssetActions/index.tsx#L21)
- any other ideas?

Reply:
I think that we can run with this assumption , but just for now. This is a compromise. Ideally in some future we actually show all the services.
> if there's a compute service there should be no download possible.